notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>The "Before you begin" section in cloud.google.com tutorials is reproduced in notebook form in the "Set up your GCP project" cell below, so you should delete this:</p><p> </p><h3>Follow before you begin in Guide</h3><p> </p><p>{ add link to any online before you begin tutorial on the product }</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>